src/reducers/listReducer.js

```diff
@@ -107,6 +111,15 @@ export default (name = '', transformer = null, comparator = null) => {
       }
     }
 
+    if (result.sort) {
+      return {
+        ...state,
+        sequence: ++state.sequence,
+        list: sort(state.list, result.sort),
```

Comment:
This should be manipulating the `transformedList`. The idea is that there is the `list` (the actual data) and the list that we use to display, the `transformedList` Right now the transformer transforms just one entry at a time rather than the whole list. If we are going to augment the reducer to do sorting, it we likely should add a `listTransformer` or some such that manages the ordering/filtering of the whole list. Then the UI just gets and displays the transformedList.